src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/CoreNode.cs

```diff
@@ -175,6 +175,16 @@ public CoreNode OverrideService(Action<IServiceCollection> serviceToOverride)
             return this;
         }
 
+        /// <summary>
+        /// Removes all the peer connectors from the node.
+        /// </summary>
+        /// <returns>This node.</returns>
+        public CoreNode WithNoConnectors()
```

Comment:
We have too many of these methods, the tests are less and less natural and more and more synthetic.
do we really really need this?